Documentation/Contributing/Roadmap.md

```diff
@@ -23,67 +26,106 @@ The Mixed Reality Toolkit will require Unity 2018.4.
 
 > When Unity releases an LTS (Long Term Support) product, the Mixed Reality Toolkit will update to the LTS release. MRTK will also support the latest non-beta (ex: 2019.1) tech branch version of Unity, at the time at which MRTK was released.
 
-### 2.3.0
+### 2.4.0
 
-The version 2.3.0 plan has been finalized. While some details may change as the iteration progresses, the following describes the overall plan for the next MRTK release. For the latest status of the release, please visit the [milestone page]( https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/10).
+For the latest status of the release, please visit the [milestone page]( https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/11).
 
 Status: In development
 
-Timeline: January 2029
+Timeline: March 2020
 
 Themes:
 
 - Stability
 - Developer education
-- Unity 2019.3 XR SDK support
 - User Experience
-- Iterate on mobile AR support
+- Hand Extensibility
 
 **Stability**
 
 Quality and stability are the top priority for this and all Microsoft Mixed Reality Toolkit releases. We will continue to prioritize customer and partner issues that impact the stability of MRTK components.
 
-The MRTK build and deploy tools remain a continued area of stability and quality focus.
-
 **Developer education**
 
 [Developer documentation](https://microsoft.github.io/MixedRealityToolkit-Unity) and example scenes are, like stability, an ongoing priority for the MRTK team.
 
-**Unity 2019.3 XR SDK support**
+**User Experience**
 
-XR SDK is Unity's new mixed reality platform. The Microsoft Mixed Reality Toolkit is committed to supporting our customers on this new platform. All new platform support in MRTK is expected to be delivered via XR SDK.
+- Bug fixes
+- Hololens Shell parity
+- Graduating experimental features
+- Tests to ensure features do not regress
 
-**User Experience**
+**Hand extensibility**
 
-User experience (UX) work will include:
+Hand extensibility includes refactors (non-breaking change) that to allow features, such as customizable pointing rays to be easily implemented without need for
+modifying MRTK source.
 
-- Bug fixes
-- Making it easier to add MRTK UX elements to projects
-- Bounding box and manipulator updates
-- New features
-    - Tap to place (HoloToolkit feature)
-    - Follow solver (https://github.com/microsoft/MixedRealityToolkit-Unity/issues/5713)
+### 2.5.0
```

Comment:
Again, I would say anything past 2.5 and I would probably not list them as explicit call outs. We should avoid ipsum lorem type text and strive for content that creates clarity.

Reply:
done

Reply:
Unresolving the conversation because this wasn't done (i.e. the text still exists and should be deleted)

Reply:
missed that line. it has been re-done